common/embed.cc

```diff
@@ -0,0 +1,45 @@
+#if defined(WIN32)
+#include <windows.h>
+#endif
+#include <boost/iostreams/device/mapped_file.hpp>
+#include <boost/filesystem.hpp>
+#include "nextpnr.h"
+#include "embed.h"
+
+NEXTPNR_NAMESPACE_BEGIN
+
+#if defined(EXTERNAL_CHIPDB_ROOT)
+
+const void *get_chipdb(const std::string &filename) {
+    static std::map<std::string, boost::iostreams::mapped_file> files;
+    if (!files.count(filename)) {
+        std::string full_filename = EXTERNAL_CHIPDB_ROOT "/" + filename;
+        if (boost::filesystem::exists(full_filename))
+            files[filename].open(full_filename, boost::iostreams::mapped_file::priv);
+    }
+    if (files.count(filename))
+        return files.at(filename).data();
+    return nullptr;
+}
+
+#elif defined(WIN32)
+
+const void *get_chipdb(const std::string &filename) {
+    HRSRC rc = ::FindResource(NULL, filename.c_str(), RT_RCDATA);
```

Comment:
I think the old code was using NULL, but can you change that to nullptr for consistency with other code in ths file?

Reply:
Fixed